libs/wingsdk/src/target-tf-aws/api.ts

```diff
@@ -414,6 +415,9 @@ class WingRestApi extends Construct {
       stageName: STAGE_NAME,
       deploymentId: this.deployment.id,
     });
+
+    //should be exported from here, otherwise won't be mapped to the right token
+    this.url = this.stage.invokeUrl;
```

Comment:
@skorfmann I am okay with this as a point solution to the `api.url` issue, but that's a more general problem with CDKTF.

Do we have a CDKTF issue to associate with this? Any chance you can pick this up upstream?